motor/motor_gridfs.py

```diff
@@ -42,6 +42,13 @@ class AgnosticGridOutCursor(AgnosticCursor):
 
     def next_object(self):
         """**DEPRECATED** - Get next GridOut object from cursor."""
+        warnings.warn(
```

Comment:
Won't this raise two duplicate warnings now?

Reply:
Good catch, added a comment instead.